### ISSUE TITLE
modals feature

### ISSUE THREAD

**Issue by spyoungtech** [spyoungtech/behave-webdriver#4]
modals feature requires implementation

```
  tests\experimental-features\modals.feature:9  Test if alert is opened accepted
  tests\experimental-features\modals.feature:17  Test if alert is opened & dismissed
  tests\experimental-features\modals.feature:24  Test if confirm is canceled
  tests\experimental-features\modals.feature:34  Test if confirm is accepted
  tests\experimental-features\modals.feature:43  Test if prompt is opened & dismissed
  tests\experimental-features\modals.feature:53  Test if prompt is accepted
  tests\experimental-features\modals.feature:62  Test if prompt has text entered
```

**Comment by spyoungtech** [comment]
Added the functionality in the features/modals branch. Although the CI build appears to pass... Experiences with running the test locally suggests that some caution may need to be taken, because if a test there fails, it may cause subsequent tests to fail. For example, if an alert is left open on the driver, subsequent tests will fail. May want to look at adding cleanups, just in case.

**Comment by spyoungtech** [comment]
closed per #29
